x-pack/test/detection_engine_api_integration/utils/index.ts

```diff
@@ -80,6 +80,16 @@ export * from './wait_for';
 export * from './wait_for_alert_to_complete';
 export * from './wait_for_event_log_execute_complete';
 export * from './wait_for_rule_status';
+export * from './prebuilt_rules/create_prebuilt_rule_saved_objects';
+export * from './prebuilt_rules/install_prebuilt_rules_and_timelines';
+export * from './get_simple_rule_update';
+export * from './get_simple_ml_rule_update';
+export * from './get_simple_rule_as_ndjson';
+export * from './rule_to_ndjson';
+export * from './delete_rule';
+export * from './get_query_signal_ids';
+export * from './get_query_signals_ids';
+export * from './get_signals_by_ids';
```

Comment:
Backport note: additional exports not from [original PR](https://github.com/elastic/kibana/pull/168058/files#diff-e68b60ed4e07497bd6764360ded2c7b9869b4e19d92390d63e0316e19a2dd80fR31-R36).